manifests/config/base/kustomization.yaml

```diff
@@ -25,4 +25,6 @@ patchesStrategicMerge: []
 # add this line to set model-server endpoint to kepler
 # - ./patch/patch-model-server-kepler-config.yaml
 # add this line to change log level for debugging
-# - ./patch/patch-debug.yaml
\ No newline at end of file
+# - ./patch/patch-debug.yaml
+# add this line to get high metric granularity (3s interval)
+# - ./patch/patch-high-granularity.yaml
```

Comment:
is this file coming next?

Reply:
@rootfs good catch. I have added the file